README.md

```diff
@@ -11,21 +11,6 @@ The DOC file has some explanations about the structure & features.
 ```
 
 
-# Dev Setup:
```

Comment:
I'd leave these instructions at the top as the secrets/local_settings must be put in by hand before running vagrant up.